docs/source/trainer/using_the_trainer.rst

```diff
@@ -461,6 +461,25 @@ points during training and (2) load them back to resume training later.
 
     The :doc:`checkpointing` guide.
 
+Gradient Accumulation
+~~~~~~~~~~~~~~~~~~~~~
+
+Composer supports gradient accumulation, which allows training arbitrary
+logical batch sizes on any hardware by breaking the batch into ``grad_accum``
+different microbatches.
+
+.. code:: python
+
+    from composer import Trainer
+
+    trainer = Trainer(
+        ...,
+        grad_accum=2,
+    )
+
+If ``grad_accum=auto``, Composer will try to automatically determine the 
```

Comment:
Added a bit more description. Not super sure how much detail we want to go into -- the core idea is pretty simple: start with grad_accum=1, and double whenever a CUDA OOM is caught. A lot of the subtlety (and long discussions) come from being careful with multi-rank edge cases, but I don't think that's super interesting / worth putting in docs

Imo the "cool" part is (1) being able to run it on any hardware and see it work out of the box and (2) speedups when using things like progressive image resizing (since you can decrease grad_accum and go faster at start of training). I briefly mentioned this in the docs, though really a blog post seems like a better vehicle (which I'll sync with Kobie on when he's feeling better)